### ISSUE TITLE
Link color advice could lead to accessibility issue

### ISSUE THREAD

**Issue by danielnixon** [edwardtufte/tufte-css#144]
This issue is regarding this paragraph discussing link styles:

> As always, these design choices are merely one approach that Tufte CSS provides by default. Other approaches, such as changing color on click or mouseover, or using highlighting or color instead of underlining to denote links, could also be made to work. The goal is to make sentences readable without interference from links, as well as to make links immediately identifiable even by casual web users.

Specifically:

> using highlighting or color instead of underlining to denote links

My concern is that a reader may follow this advice without being aware of WCAG 2.0 [1.4.1 Use of color](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color):

> Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. (Level A)

It might be best to simply remove this advice.

**Comment by danielnixon** [comment]
Noting the irony that we're discussing this issue on a platform that is itself guilty of this very sin.

**Comment by danielnixon** [comment]
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F73:

> The objective of this failure is to avoid situations in which people who cannot perceive color differences cannot identify links (when people with color vision can identify links). Link underlines or some other non-color visual distinction are required (when the links are discernible to those with color vision).
>
> While some links may be visually evident from page design and context, such as navigational links, links within text are often visually understood only from their own display attributes. Removing the underline and leaving only the color difference for such links would be a failure because there would be no other visual indication (besides color) that it is a link.

**Comment by daveliepmann** [comment]
Thanks for pointing this out. You're not wrong. However, I want to preserve the "Other approaches...could also be made to work." sentence, preferably with the "such as X or Y" verbiage in the middle. Pointing people to accessible alternatives is, I think, just as important as not pointing them to inaccessible alternatives. 

Could you propose some phrasing in this thread? Once we are in the right ballpark, a PR would be very welcome.
